### PR TITLE
VPN-3703 - Implement What's New messsage for 2.13

### PR DESCRIPTION
**Pull request by brizental**
Figma: https://www.figma.com/file/Disx2dX7tw81NQIN39BrYo/In-app-Messaging?node-id=1515%3A5328&t=qaBi1ZXEt4QIyapE-0

### CODE REVIEW

File: addons/message_whats_new_v2.13/manifest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          {
+            "id": "l_4",
+            "content": "On Android, you can now connect and disconnect to the VPN using the quick actions in the notification bar."
+          },
+          {
+            "id": "l_5",
+            "content": "New multi-hop and app exclusion tutorials."
+          },
+          {
+            "id": "l_4",
```
Comment (brizental) [inline addons/message_whats_new_v2.13/manifest.json:41]:
```suggestion
            "id": "l_6",
```

File: addons/message_whats_new_v2.13/manifest.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        "id": "c_1",
+        "type": "ulist",
+        "content": [
+          {
+            "id": "l_2",
+            "content": "When running a speed test, you’ll now be able to see upload as well as download speed."
+          },
+          {
+            "id": "l_3",
+            "content": "You can now see the IP address of the server you are connected by tapping the info button."
```
Comment (flodolo) [inline addons/message_whats_new_v2.13/manifest.json:30]: Question for content: isn't this missing a `to` (`…you are connected to by…`)?

File: addons/message_whats_new_v2.13/manifest.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          },
+          {
+            "id": "l_4",
+            "content": "On Android, you can now connect and disconnect to the VPN using the quick actions in the notification bar."
```
Comment (flodolo) [inline addons/message_whats_new_v2.13/manifest.json:34]: Another question for content: this sounds off to me (`disconnected to the VPN`)?

File: addons/message_whats_new_v2.13/manifest.json (hunk 2 above)
Comment (brizental) [reply addons/message_whats_new_v2.13/manifest.json:30]: cc @chrisrooneys

File: addons/message_whats_new_v2.13/manifest.json (hunk 3 above)
Comment (brizental) [reply addons/message_whats_new_v2.13/manifest.json:34]: cc @chrisrooneys

File: addons/message_whats_new_v2.13/manifest.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.13",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.13.0",
+    "max_client_version": "2.13.9"
+  },
```
Comment (bakulf) [inline addons/message_whats_new_v2.13/manifest.json:9]: if we are not 100% sure of the translations: `translatable: false`

**Review by bakulf — APPROVED** [review]
R+ for code.

File: addons/message_whats_new_v2.13/manifest.json (hunk 4 above)
Comment (brizental) [reply addons/message_whats_new_v2.13/manifest.json:9]: Let's wait for Chris's responses... If we are still undure after that I'll add the flag.

**Review by MattLichtenstein — APPROVED** [review]

File: addons/message_whats_new_v2.13/manifest.json (hunk 2 above)
Comment (brizental) [reply addons/message_whats_new_v2.13/manifest.json:30]:
```suggestion
            "content": "You can now see the IP address of the server you are connected to by tapping the info button."
```

File: addons/message_whats_new_v2.13/manifest.json (hunk 3 above)
Comment (brizental) [reply addons/message_whats_new_v2.13/manifest.json:34]:
```suggestion
            "content": "On Android, you can now turn the VPN on or off using the quick actions in the notification bar."
```

**Review by flodolo — APPROVED** [review]

**Pull request merged**